lib/pipeline/addon/utils/timezones.js

```diff
@@ -0,0 +1,1353 @@
+export let timezones = [
```

Comment:
How much of this data do you actually use?  There's 27+kb of data to load and parse here, and for example you only seem to use the first `utc` value.

Reply:
the text and utc, I will delete the others.